main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
+An *object capability system* constrains how references are obtained. You can't get one just by 
+knowing the name of a global variable or a public class. You can only get a reference via: 
 - Creation: Functions that create objects get a reference to them.
 - Construction: Constructors can endow their constructed objects with references, including inherited references. 
 - Introduction: 
   - A has references to B and C. 
-  - B and C  do *not* have references to each other
+  - B and C do *not* have references to each other
   - A sends B a reference to C. 
     - B now has a reference to C and can communicate with C. 
 
-If references can only be obtained by creation, construction, or introduction, you may have a safe system. If they can be obtained in any other way, your system is unsafe.
+If references can only be obtained by creation, construction, or introduction, you may have a safe 
+system. If they can be obtained in any other way, your system is unsafe.
 
 For more information, see [Douglas Crockford on Object Capabilities](https://frontendmasters.com/courses/good-parts-javascript-web/object-capabilities/).
 
+## Offer
+
+Offers are a structured way of describing user intent. In Zoe, an offer consists of a [proposal](#proposal) (the 
+rules under which the party wants to exercise the offer) and [payments](#payment) corresponding to what the proposal specifies as what the
+party will give if the offer is satisfied. The payments are automatically [escrowed](#escrow) by Zoe, and appropriately reallocated when
+the offer exits with either success or rejection. See [`E(Zoe).offer(invitation, proposal, paymentKeywordRecord)`](https://agoric.com/documentation/zoe/api/zoe.html#e-zoe-offer-invitation-proposal-paymentkeywordrecord).
+
+## Offer Safety
+
+Zoe guarantees offer safety. When a user makes an [offer](#offer) and it is [escrowed](#escrow) with Zoe, Zoe guarantees that 
+the user either gets what they said they wanted, or gets back (gets a refund) what they originally offered and escrowed.
+
 ## Payment
-Payments hold [amounts](#amounts) of certain assets 
-issued by [Mints](#mint). Specifically amounts that are *in transit* from one party to another. 
-Amounts from payments can be deposited in [purses](#purse), but otherwise, the entire amount is 
-available when the payment is transferred. Payments can be converted to [Purses](#purse). All contents 
-of a purse must be of the same [brand](#brand).
-
-See the [ERTP Guide's Payments section](/ertp/guide/purses-and-payments.md#payments)
-and the [ERTP API's Payments section](/ertp/api/payment.md#payment-getallegedbrand) 
-for more information.
-
-## Presence
-A local version of a remote object that serves as a proxy for the remote object. 
+
+Payments hold assets issued by [Mints](#mint). Specifically assets intended for transfer 
+from one party to another. All assets of a payment must be of the same [brand](#brand).
```

Comment:
```suggestion
from one party to another. All assets of a payment are of the same [brand](#brand).
```
`must be` sounds like we are instructing people to not do something wrong, but there is no way in which this can be otherwise.